src/test/fuzz/multiplication_overflow.cpp

```diff
@@ -14,6 +10,14 @@
 #include <string>
 #include <vector>
 
+#if defined(__has_builtin)
+#if __has_builtin(__builtin_mul_overflow)
+#define HAVE_BUILTIN_MUL_OVERFLOW
+#endif
+#elif defined(__GNUC__)
+#define HAVE_BUILTIN_MUL_OVERFLOW
```

Comment:
nit: I understand that that was a clean commit revert. But these lines look really unneeded these days.